### PR TITLE
Some feedback suggestions on the blog post

### PR DESCRIPTION
**Pull request by stichbury**
I've made some minor style changes but this was really good. Mostly to make things a bit clearer for people new to both Kedro & Dolt. I felt the datacatalog section was a bit confusing at the start, so I added a bit more info there. Please take a look to see if that works for you.

There's a couple of bits I need some help with too, please. One is the audience for the post (I like to put it up-front so people know if they'll benefit from reading it) and the other is "what they'll learn" (a sentence or two to explain the value of reading it)

Please feel free to revise my feedback and accept/reject at will. I think it's great; very nice 🌟

### CODE REVIEW

File: blog.md [hunk 1]
```diff
@@ -342,7 +346,9 @@ $  dolt sql -q "select message from dolt_log limit 1"
 ## Conclusion
 
 The Kedro-Dolt plugin currently supports a limited set of features.
-We encourage you to [reach out on our
-Discord](https://discord.com/invite/RFwfYpu) if you are interested in a
+We encourage you to [reach out on the Kedro
+Discord channel](https://discord.com/invite/RFwfYpu) if you are interested in a
```
Comment (yetudada) [inline blog.md]:
```suggestion
Discord channel](https://discord.gg/2QvVEdVEea) if you are interested in a
```

File: blog.md [hunk 2]
```diff
@@ -8,21 +8,28 @@ authorHref: "https://www.dolthub.com/team#max"
 # Introduction
 
 We are pleased to introduce a Kedro-Dolt plugin, a collaboration between
-Quantum Black and DoltHub designed to expand the data-versioning
-abilities of data scientists and engineers.
+[Quantum Black Labs](https://www.quantumblack.com/labs) and [DoltHub](https://www.dolthub.com/) designed to expand the data-versioning
+abilities of data scientists and engineers. You will find this a useful read if you are [A WHAT? Data engineer? ML Engineer] and it'll take about 5 minutes to read, by the end of which time, you'll know how to [DO WHAT?].
```
Comment (yetudada) [inline blog.md:12]: @stichbury Is there supposed to be a link on the `[A WHAT? Data engineer? ML Engineer]` and `[DO WHAT]`?

File: blog.md (hunk 1 above)
Comment (yetudada) [reply blog.md]: This link doesn't expire.

**Comment by max-hoffman** [comment]
@stichbury Thanks for reviewing! I appreciate the feedback.

**Pull request merged**